### DISCUSSION TITLE
checking: libseccomp+headers... no

### DISCUSSION THREAD

**Discussion by eswk-im** [sylabs/singularity#1951] [Q&A]
Hi, 

I am running into an issue while running mconfig. I am trying to install a local copy of singularity as I do not have access to the sudo command in the cluster. I've installed libseccomp and my seccomp headers are located in:

/users/username/local/include 

I have two files: "seccomp.h  seccomp-syscalls.h" under /users/username/local/include  folder. 

When I run ./mconfig I get the following error: 

checking: libseccomp+headers... no

seccomp headers are required to build Singularity with seccomp support.
To disable seccomp support run mconfig using '--without-seccomp'.

I tried using the CFLAGS option but it doesn't seem to be a viable option for mconfig. 

 ./mconfig CFLAGS='-I/users/username/local/include'
Usage: mconfig [-vsS] [-P profile] [-b builddir] [-c hstcc] [-C tgtcc]
                [-V version] [-x hstcxx] [-X tgtcxx]
                [--INSTALLDIR=ARG (see below)]

How may I resolve this error?

Thank you for your help.

**Comment by dtrudg** [comment]
The `mconfig` scripts use `pkg-config` to find the libseccomp.

Make sure that the path to the `.pc` file for your installation of libseccomp is included in your `PKG_CONFIG_PATH` environment variable.
